library.properties

```diff
@@ -0,0 +1,11 @@
+name=WebMonitor
+version=0.4.1
+author=Boris Jaeger <sivar2311@gmail.com>
+maintainer=Boris Jaeger <sivar2311@gmail.com>
+sentence=Web Monitor Library for ESP8266 & ESP32
+paragraph=Access your logs via WiFi
+category=Communication
+repository=https:://github.com/sivar2311/WebMonitor.git
+includes=WebMonitor.h
+url=https:://github.com/sivar2311
+depends=AsyncTCP,ESPAsyncTCP
```

Comment:
WebMonitor is dependent to [ESPAsyncWebServer](https://github.com/me-no-dev/ESPAsyncWebServer)
and thus is dependent to [ESPAsyncTCP](https://github.com/me-no-dev/ESPAsyncTCP) on ESP8266 / [AsyncTCP](https://github.com/me-no-dev/AsyncTCP) on ESP32.

Does the Arduino Library Manger support different dependencies for different platforms?

I suggest changing the dependency to "ESPAsyncWebServer".

Reply:
done